extensions-core/histogram/src/main/java/org/apache/druid/query/aggregation/histogram/ApproximateHistogram.java

```diff
@@ -41,8 +41,8 @@ public class ApproximateHistogram
   // max size of the histogram (number of bincount/position pairs)
   int size;
 
-  public float[] positions;
-  public long[] bins;
+  protected float[] positions;
+  private long[] bins;
```

Comment:
why mix protected/private?

I don't see it wrong to have a pojo with open fields - but if thats not good; they should all be protected so that subclasses could do things

Reply:
Have made the change.